### PR TITLE
Makefile: set the url of chart repository

### PR DESCRIPTION
**Pull request by joejulian**
We need to set the url of the chart repository to get the index with the appropriate entry.

### CODE REVIEW

File: docs/stable/index.yaml [hunk 1]
```diff
@@ -1,3 +1,13 @@
 apiVersion: v1
-entries: {}
-generated: 1970-01-01T00:00:00.000000000-00:00
+entries:
```
Comment (joejulian) [inline docs/stable/index.yaml]: please revert these changes to this and the staging/index.yaml

File: docs/staging/index.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 apiVersion: v1
 entries: {}
-generated: 1970-01-01T00:00:00.000000000-00:00
+generated: 2019-07-02T15:02:08.769246+02:00
```
Comment (joejulian) [inline docs/staging/index.yaml]: revert

Comment (joejulian) [reply docs/staging/index.yaml]: And delete the tarball

File: docs/stable/index.yaml [hunk 3]
```diff
@@ -1,3 +0,0 @@
-apiVersion: v1
```
Comment (joejulian) [inline docs/stable/index.yaml]:
```
apiVersion: v1
entries: {}
generated: 1970-01-01T00:00:00.000000000-00:00
```
for both docs/stable/index.yaml and docs/staging/index.yaml

This ensures that if either are unable to be generated (no charts, for instance) an empty index file will still exist.

File: Makefile [hunk 4]
```diff
@@ -50,8 +52,7 @@ $(STABLE_TARGETS) $(STAGING_TARGETS): $(TMPDIR)/.helm/repository/local/index.yam
 %/index.yaml: $(STABLE_TARGETS) $(STAGING_TARGETS)
 %/index.yaml: $(TMPDIR)/.helm/repository/local/index.yaml
 	@mkdir -p $(patsubst %/index.yaml,%,$@)
-	$(HELM) --home $(TMPDIR)/.helm repo index $(patsubst %/index.yaml,%,$@)
```
Comment (gpaul) [inline Makefile]: Could we get `mesosphere` from `git remote origin`? I'd like to use this on my fork. Something like:

```
--url=https://$(shell git remote get-url origin | cut -d':' -f2 | cut -d'/' -f1).github.io/charts/$(patsubst...)
```

Comment (gpaul) [reply Makefile]: higher up in  this file, could we have:
```
-	@git remote add publish git@github.com:mesosphere/charts >/dev/null 2>&1 || true
+	@git remote add publish $(shell git remote get-url origin) >/dev/null 2>&1 || true
```

Comment (gpaul) [reply Makefile]: I'll fix these in my next PR, no worries.
